packages/core/src/builder.class.ts

```diff
@@ -2630,16 +2630,19 @@ export class Builder {
         }
       }
     }
-    if (this.preview) {
-      queryParams.preview = 'true';
-    }
-    const hasParams = Object.keys(queryParams).length > 0;
 
     // TODO: option to force dev or qa api here
     const host = this.host;
 
     const keyNames = queue.map(item => encodeURIComponent(item.key!)).join(',');
 
+    const isSymbol = keyNames.includes('symbol');
+
+    if (this.preview && !isSymbol) {
```

Comment:
rather than checking for 'symbol' should we check if the `previewingModel === modelPassed`?

```suggestion
    if (this.preview && this.previewingModel === parseLogic(keyNames)) { // parse to take out anything before %
```

or

```suggestion
    if (this.preview && this.previewingModel === queue[0].model)) {
```

Reply:
@sidmohanty11 the issue here is that this.previewingModel is always coming as 'page' in both the page call as well as the symbol call. While the model in the queue is coming as 'page' for page call and 'symbol' for symbol call

Reply:
is that not what we want? `previewingModel` for page will be `page` === `queue[0].model` where we want preview=true and later for symbol call `previewingModel` is still `page` which is not equal to `symbol` model where we shouldn't add preview right or am I missing something?

Reply:
Right ! makes sense. I got confused. I have pushed the changes